### ISSUE TITLE
Rowspanned Cells between pages extend border and fill color

### ISSUE THREAD

**Issue by rUI-Bullet** [bpampuch/pdfmake#928]
```js
var dd = {
    pageSize: 'A5',
    pageOrientation: 'landscape',
    pageMargins: [ 40, 60, 40, 60 ],
	content: [
		{text: 'Title', style: 'header'},
		{
			table: {
				headerRows: 1,
				body: [
				    [{text: '#', style: 'tableHeader'}, {text: 'Lorem Ipsum', style: 'tableHeader'}],
				    ['1', 'Lorem Ipsum Dolor Sit Amet'],
				    ['2', 'Lorem Ipsum Dolor Sit Amet'],
				    ['3', 'Lorem Ipsum Dolor Sit Amet'],
				    ['4', 'Lorem Ipsum Dolor Sit Amet'],
				    ['5', 'Lorem Ipsum Dolor Sit Amet'],
				    ['6', 'Lorem Ipsum Dolor Sit Amet'],
				    ['7', 'Lorem Ipsum Dolor Sit Amet'],
				    ['8', 'Lorem Ipsum Dolor Sit Amet'],
				    ['9', 'Lorem Ipsum Dolor Sit Amet'],
				    ['10', 'Lorem Ipsum Dolor Sit Amet'],
				    ['12', 'Lorem Ipsum Dolor Sit Amet'],
				    ['13', {rowSpan: 5, fillColor: 'red', text : 'Lorem Ipsum Dolor Sit Amet \n Lorem Ipsum Dolor Sit Amet \n Lorem Ipsum Dolor Sit Amet. \n Lorem Ipsum Dolor Sit Amet \n Lorem Ipsum Dolor Sit Amet' }],
				    ['14', ''],
				    ['15', ''],
				    ['16', ''],
				    ['17', ''],
				    ['18', 'Lorem Ipsum Dolor Sit Amet'],
				    ['19', 'Lorem Ipsum Dolor Sit Amet'],
				    ['20', 'Lorem Ipsum Dolor Sit Amet']
				]
			}
		}
	],
	styles: {
		header: {
			fontSize: 18,
			bold: true,
			margin: [0, 0, 0, 10]
		},
		tableHeader: {
			bold: true,
			fontSize: 13
		}
	}
}
```

In the given example, if the text in a rowspanned cell passes to another page, the vertical borders of the table extend to the end of the page, as the fillColor of that cell.

![e03](https://cloud.githubusercontent.com/assets/20336574/24361357/a9a4aeb0-1301-11e7-88ac-3c51f599c772.png)

**Comment by rUI-Bullet** [comment]
Hello,
Any news on this issue?
Thanx

**Comment by rUI-Bullet** [comment]
Hello,
Do you have any idea if this issue is going to be corrected soon?
It has been tagged as Bug almost two months ago.
Thanx

**Comment by yshinjp** [comment]
Hello,

I also wait fixes about this.
Does someone know which part should be changed ? I'm ready to try fix this.

Thanks.

**Comment by novoo** [comment]
Hi,
Any news on this issue? Any know workarounds?
Thanks

**Comment by ShaneShipston** [comment]
I am also experiencing this issue.

**Comment by Flynnp25** [comment]
Hi,
Is there an update on this?

**Comment by PI-YWT** [comment]
Is there any news on this?

**Comment by AnibalDBXD** [comment]
Hi,
Is there an update on this?

**Comment by mojidul-dev-pwd** [comment]
Hello,
Is there any update on this?

**Comment by tranhongs0n** [comment]
Hello,
Is there any update on this?

**Comment by liborm85** [comment]
Fix released in version 0.2.15.
